lib/pbench/cli/agent/commands/results/move.py

```diff
@@ -0,0 +1,257 @@
+    def execute(self, single_threaded: bool, copy_only: bool = False) -> int:
+        runs_copied = 0
+        failures = 0
+        no_of_tb = 0
+
+        with tempfile.TemporaryDirectory(
+            dir=self.config.pbench_tmp, prefix="pbench-move-results."
+        ) as temp_dir:
+            for dirent in self.config.pbench_run.iterdir():
+                if not dirent.is_dir():
+                    continue
+                if dirent.name.startswith("tools-") or dirent.name == "tmp":
+                    continue
+
+                no_of_tb += 1
+                result_dir = dirent
+
+                try:
+                    mrt = MakeResultTb(
+                        result_dir,
+                        temp_dir,
+                        self.context.controller,
+                        self.config,
+                        self.logger,
+                    )
+                except (
+                    NotADirectoryError,
+                    FileNotFoundError,
+                    MakeResultTb.AlreadyCopied,
+                    MakeResultTb.BenchmarkRunning,
+                ) as exc:
+                    self.logger.error(str(exc))
+                    failures += 1
+                    continue
+
+                try:
+                    result_tb_name, result_tb_len, result_tb_md5 = mrt.make_result_tb(
+                        single_threaded=single_threaded
+                    )
+                except BadMDLogFormat as exc:
+                    self.logger.warning(str(exc))
+                    failures += 1
+                    continue
+                except FileNotFoundError as exc:
+                    self.logger.error(str(exc))
+                    failures += 1
+                    continue
+                except RuntimeError as exc:
+                    self.logger.warning("Error encountered making tar ball, '%s'", exc)
+                    failures += 1
+                    continue
+                except Exception as exc:
+                    self.logger.error(
+                        "Unexpected error occurred making tar ball for '%s', '%s'",
+                        result_dir,
+                        exc,
+                    )
+                    failures += 1
+                    continue
+
+                try:
+                    crt = CopyResultTb(
+                        self.context.controller,
+                        result_tb_name,
+                        result_tb_len,
+                        result_tb_md5,
+                        self.config,
+                        self.logger,
+                    )
+                    crt.copy_result_tb(self.context.token)
+                except (CopyResultTb.FileUploadError, RuntimeError) as exc:
+                    self.logger.error(
+                        "Error uploading file, '%s', '%s'", result_tb_name, exc
+                    )
+                    failures += 1
+                    # We don't know what this operation failed; regardless,
```

Comment:
"why this operation failed"??? And it's repeated multiple times below. Also, the "regardless, given [...] this tar ball remotely," probably isn't really useful, and the longer comment is just harder to read. Maybe just "We don't know why the operation failed, but trying to copy another tarball isn't likely to succeed."